fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/FetchResourceChangesDAO.java

```diff
@@ -64,6 +79,21 @@ public FetchResourceChangesDAO(IDatabaseTranslator tx, String schemaName, int re
      */
     public List<ResourceChangeLogRecord> run(Connection c) throws FHIRPersistenceException {
         List<ResourceChangeLogRecord> result = new ArrayList<>();
+        
+        Instant before = null;
```

Comment:
I feel like we also talked about possibly taking `_before` as query parameter (similar to `_since` but for paging through results in the opposite direction).  I don't know if its needed or not, but wanted to see if you'd thought about that one.

Reply:
Yeah, less necessary now because you can switch the sort order, so _since behaves that way. If we add _before, then we'd need to make the semantics the same for _since and _before regardless of sort order, which may not be a bad thing because I was struggling about using _since for both directions. Gonna chew on this.

Reply:
Per our conversation I've now included support for both `_before` and `_since` - both are supported for the lastUpdated ASC and DESC ordering.

Reply:
Updated the Conformance.md doc accordingly.